pkg/virt-controller/services/template.go

```diff
@@ -168,6 +169,16 @@ func CPUModelLabelFromCPUModel(vmi *v1.VirtualMachineInstance) (label string, er
 	return
 }
 
+func CPUFeatureLabelsFromCPUFeatures(vmi *v1.VirtualMachineInstance) []string {
+	var labels []string
+	if vmi.Spec.Domain.CPU != nil && vmi.Spec.Domain.CPU.Features != nil {
+		for _, feature := range vmi.Spec.Domain.CPU.Features {
+			labels = append(labels, NFD_CPU_FEATURE_PREFIX+feature.Name)
```

Comment:
Only labels with policy `require` should be converted to labels (optional, disable and force should not). The same applies to `forbid`, but those must create a negative selector.

Reply:
good point